fluent-plugin-enhance-k8s-metadata/fluent-plugin-enhance-k8s-metadata.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'concurrent-ruby', '~> 1.1'
   spec.add_runtime_dependency 'fluentd', ['>= 0.14.10', '< 2']
-  spec.add_runtime_dependency 'kubeclient', '~> 4.4.0'
+  spec.add_runtime_dependency 'kubeclient', '~> 4.4'
```

Comment:
what does this file do after pinning a static version?

Reply:
this allows `>= 4.4 and `< 5.0`